app/components/channel_drawer/channel_drawer.js

```diff
@@ -180,10 +192,9 @@ export default class ChannelDrawer extends PureComponent {
         }
     };
 
-    selectChannel = (channel) => {
+    selectChannel = (channel, currentChannelId) => {
```

Comment:
Great idea to pass down the currentChannelId from whatever is calling this. I was struggling to come up with a way to do it without having the drawer rerender whenever the channel changes